packages/augur-ui/src/modules/market/components/market-view/market-view.tsx

```diff
@@ -504,6 +502,7 @@ export default class MarketView extends Component<
     }
 
     let outcomeOrderBook = this.EmptyOrderBook;
+    const orderbookLoading = !!!orderBook;
```

Comment:
Looks like you may mean `!orderbook`

Reply:
it's an object, converting to boolean and checking if false.

Reply:
actually, could just use !orderBook. I'll make the change